scalableminds/webknossos
# Call webknossos-worker Generate Mesh File Job

Call webknossos-worker job to create mesh file from an existing segmentation layer.

Note that this PR does not yet include a UI for this, which will presumably be added later after #5346 is done.

### Steps to test:
- set up https://github.com/scalableminds/webknossos-worker/pull/15
- set jobsEnabled = true
- call `/api/jobs/run/computeMeshFile/sample_organization/l4_sample?layerName=segmentation&mag=4-4-1&agglomerateView=agglomerate_view_20` (replace by values pointing to existing data. agglomerateView is optional)
- job should succeed, dataset should have meshfile

------
